cunumeric/coverage.py

```diff
@@ -150,18 +154,23 @@ def wrapper(*args: Any, **kwargs: Any) -> Any:
                 location=location,
                 implemented=False,
             )
+            if is_method:
+                args = (args[0].__array__(),) + args[1:]
             return func(*args, **kwargs)
```

Comment:
Would `isinstance` check with `types.MethodType` work instead of having to pass an explicit flag?